spec/controllers/api_v1_users_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::UsersController, type: :controller do
+  describe 'API' do
+    describe 'GET /users/998' do
+      it 'works!' do
+        create(:general_user)
+        get 'show', params: { id: '998' }, format: 'json'
+        expect(response.status).to eq(200)
+        json = JSON.parse(response.body);
+        keys = ['id', 'username', 'display_name', 'biography', 'slides_count']
+        keys.each do |k|
+          expect(json.has_key?(k)).to eq(true)
```

Comment:
__[rubocop]__
* Style/PreferredHashMethods: Use `Hash#key?` instead of `Hash#has_key?`. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-key)

<!-- Comment by [SideCI](https://sideci.com) -->